src/components/CopyToClipboard/CopyToClipboard.tsx

```diff
@@ -0,0 +1,30 @@
+import copyText from 'lib/copyText'
+import * as React from 'react'
+
+type Props = {
+  text: string
+  children: React.ReactNode
+  role: 'button' | 'option'
+  className?: string
+  onCopy?: () => void
+}
+
+//TODO: if this copyText is not passed anywhere else then just remove this param
```

Comment:
I'm not sure I understand this comment, `copyText` is not passed as a parameter, right?